<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndSqlite</class>
 <widget class="QWidget" name="wndSqlite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wndSqlite</string>
  </property>
  <widget class="QLineEdit" name="id_edit_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_return_table">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>返回全表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_delete">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>170</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_ascend">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按id升序排列</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_add">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>添加记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_query">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_descend">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按id降序排列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_commit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提交修改</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="id_tab_list">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
